packages/web-components/fast-foundation/src/tabs/tabs.ts

```diff
@@ -183,13 +183,14 @@ export class Tabs extends FASTElement {
                 if (this.activeTabIndex === index) {
                     this.activetab = tab;
                 }
-            } else {
-                this.showActiveIndicator = false;
             }
             tab.style[gridProperty] = `${index + 1}`;
             !this.isHorizontal()
                 ? tab.classList.add("vertical")
                 : tab.classList.remove("vertical");
+            this.showActiveIndicator = !!this.tabs.find(
```

Comment:
With the change, the active indicator will now show up as soon as at least _one_ tab is focusable

Reply:
The query seems unnecessary since we are already iterating through the tabs in the forEach loop immediately beforehand.  Could we not just set ` this.showActiveIndicator = false;` before the forEach loop and remove the trailing else that sets it to false as you have already done?

Reply:
Indeed, applying the fix right now!